src/node/rpc/test/member_voting_test.cpp

```diff
@@ -821,7 +821,7 @@ DOCTEST_TEST_CASE("Propose raw writes")
   logger::config::level() = logger::INFO;
   DOCTEST_SUBCASE("insensitive tables")
   {
-    const auto n_members = 10;
+    const auto n_members = 3;
```

Comment:
Incremental store setup is kinda expensive, ~10 seconds in all for this test. Down to 3 members, it's consistently <1s, and I don't think it changes what we're testing functionally here.

Once this is done, ctest -L unit is consistently <3s, and there's no point using -j anymore. Small win, but still a win.